src/test-utils/browser-automation/browser-driver.js

```diff
@@ -93,26 +85,50 @@ export default class BrowserDriver {
   stopBrowser() {
     return Promise.resolve()
       .then(_ => this.page.waitFor(1000))
-      .then(_ => this.browser.close())
-      .catch(error => {
-        console.error(addColor(ERR_AUTOMATION, COLOR.BRIGHT_RED)); // eslint-disable-line
-        throw error;
-      });
+      .then(_ => this.browser.close());
   }
 
-  startServer(config = {}) {
-    this.child = this.execFile(
-      config.process || DEFAULT_CONFIG.process,
-      config.parameters || DEFAULT_CONFIG.parameters,
-      config.options || DEFAULT_CONFIG.options,
-      (err, stdout) => {
-        if (err) {
-          this.console.error(err);
-          return;
+  startServer(config = {}, maxRetryTimes = 30) {
+    const newConfig = Object.assign({}, DEFAULT_CONFIG, config);
+    return new Promise((resolve, reject) => {
+      log.log({
+        message: `Binding to port: ${newConfig.port}`,
+        color: COLOR.YELLOW
+      })();
+      const timeout = setTimeout(() => { // eslint-disable-line
```

Comment:
Just do a /* global setTimeout */ at the top of the file and drop the lint disable here.